examples/azure/terraform-azure-automationaccount-example/variables.tf

```diff
@@ -0,0 +1,125 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ENVIRONMENT VARIABLES
+# Define these secrets as environment variables
+# Note that the TF_VAR_AUTOMATION_ACCOUNT_CLIENT_ID and TF_VAR_AUTOMATION_ACCOUNT_CLIENT_PASSWORD 
+# variables represent a service principal in AAD
+# In addition to created a service prinicpal for the automation run as connection, a certificate 
+# must be uploaded to the service principal as a secret.
+# The same certificate must also be uplaoded into the Azure Automation Account via Terraform 
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ARM_CLIENT_ID
+# ARM_CLIENT_SECRET
+# ARM_SUBSCRIPTION_ID
+# ARM_TENANT_ID
+# ARM_ENVIRONMENT
+# AZURE_ENVIRONMENT
+# TF_VAR_cloud_environment
+# TF_VAR_location
+# TF_VAR_ARM_SUBSCRIPTION_ID
+# TF_VAR_ARM_TENANT_ID
+# TF_VAR_automation_run_as_certificate_thumbprint
+# TF_VAR_client_id
+# TF_VAR_client_secret
+# TF_VAR_automation_run_as_certificate_thumbprint
+# TF_VAR_automation_run_as_certificate_base64
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "ARM_SUBSCRIPTION_ID" {}
+variable "ARM_TENANT_ID" {}
```

Comment:
I don't think these two variables need to be terraform input vars, right? Since they aren't referenced directly in the module?